### PR TITLE
Add Teclis and Morfent

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: mods/seasonal/statuses.js [hunk 1]
```diff
@@ -654,6 +654,31 @@ let BattleStatuses = {
...
...<span class="col iconcol"><span class="picon" style="background: transparent url(&quot;//play.pokemonshowdown.com/sprites/smicons-sheet.png?a4&quot;) no-repeat scroll -400px -210px"></span></span> <span class="col pokemonnamecol" style="white-space: nowrap"><a href="https://pokemonshowdown.com/dex/pokemon/gengar" target="_blank" rel="noopener">Gengar</a></span> <span class="col typecol"><img src="https://play.pokemonshowdown.com/sprites/types/Ghost.png" alt="Ghost" height="14" width="32"><img src="https://play.pokemonshowdown.com/sprites/types/Poison.png" alt="Poison" height="14" width="32"></span> <span style="float: left ; min-height: 26px"><span class="col abilitycol">Cursed Body</span><span class="col abilitycol"></span></span><span style="float: left ; min-height: 26px"><span class="col statcol"><em>HP</em><br>60</span> <span class="col statcol"><em>Atk</em><br>65</span> <span class="col statcol"><em>Def</em><br>60</span> <span class="col statcol"><em>SpA</em><br>130</span> <span class="col statcol"><em>SpD</em><br>75</span> <span class="col statcol"><em>Spe</em><br>110</span> <span class="col bstcol"><em>BST<br>500</em></span> </span></li><li style="clear: both"></li></ul>`);
```
Comment (ghost) [inline mods/seasonal/statuses.js:689]: dw bout that, the code works
screenshot 77

**Comment by ghost** [comment]
In case you are wondering, i couldn’t code the reverse foresight into the move, it just wouldn’t work

File: mods/seasonal/random-teams.js [hunk 2]
```diff
@@ -371,6 +377,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: 'GO INDA',
 				evs: {atk: 252, hp: 248, spe: 8}, nature: 'Adamant',
 			},
+			'Teclis': {
+				species: 'Gallade', ability: 'Intimidate', item: 'Galladite', gender: 'M',
+				moves: ['Sacred Sword', 'Psycho Cut', 'Knock Off'],
+				signatureMove: 'Zekken',
+				evs: {atk: 252, def: 4, spe: 252}, nature: 'Timid',
```
Comment (HoeenCoder) [inline mods/seasonal/random-teams.js]: They asked for Jolly in thread and shiny too

Comment (ghost) [reply mods/seasonal/random-teams.js]: Ah alright

File: mods/seasonal/moves.js [hunk 3]
```diff
@@ -904,6 +908,47 @@ let BattleMovedex = {
...
+		priority: 0,
+		flags: {},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, 'Rock Polish', source);
+			this.add('-anim', source, 'Let\'s Snuggle Forever', target);
+		},
+		onHit: function () {
+			let messages = ["SPL players don't want you to know about this secret",
+				"A Portuguese RBYer seduces e-girls with this amazing trick",
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: Vetoing this phrase, remove it.

File: mods/seasonal/random-teams.js [hunk 4]
```diff
@@ -389,6 +413,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: 'Ultra Succ',
 				evs: {atk: 252, hp: 252, spd: 4}, nature: 'Adamant',
 			},
+			'The Leprechaun': {
+				species: 'Bronzong', ability: 'Steelworker', item: 'Life Orb', gender: 'N',
+				moves: ['Photon Geyser', 'Precipice Blades', 'Diamond Storm'],
+				signatureMove: 'Gyro Ballin\'',
+				evs: {atk: 252, hp: 252, spd: 4}, nature: 'Adamant',
```
Comment (HoeenCoder) [inline mods/seasonal/random-teams.js]: its 4 def not spd, also hp -> atk -> def

0 spe IVs

Brave nature

**Pull request merged**